File: CS/PrintingMultipleBarCharts/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts.Native;
using DevExpress.XtraReports.UI;
// ...

namespace PrintingMultipleBarCharts {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            ((IChartContainer)report.xrChart1).Chart.Assign(((IChartContainer)chartControl1).Chart);
            ((IChartContainer)report.xrChart2).Chart.Assign(((IChartContainer)chartControl2).Chart);
            ((IChartContainer)report.xrChart3).Chart.Assign(((IChartContainer)chartControl3).Chart);
            ((IChartContainer)report.xrChart4).Chart.Assign(((IChartContainer)chartControl4).Chart);
            ReportPrintTool printTool = new ReportPrintTool(report);
            printTool.ShowPreview();
        }
    }
}

File: VB/PrintingMultipleBarCharts/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts.Native
Imports DevExpress.XtraReports.UI
' ...

Namespace PrintingMultipleBarCharts
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim report As New XtraReport1()

			CType(report.xrChart1, IChartContainer).Chart.Assign((CType(chartControl1, IChartContainer)).Chart)
			CType(report.xrChart2, IChartContainer).Chart.Assign((CType(chartControl2, IChartContainer)).Chart)
			CType(report.xrChart3, IChartContainer).Chart.Assign((CType(chartControl3, IChartContainer)).Chart)
			CType(report.xrChart4, IChartContainer).Chart.Assign((CType(chartControl4, IChartContainer)).Chart)



			Dim printTool As New ReportPrintTool(report)

			printTool.ShowPreview()
		End Sub

	End Class
End Namespace